examples/db_connector/basic.py

```diff
@@ -24,7 +24,7 @@
 
 
 if __name__ == "__main__":
-    if get_auto_arg():
+    if is_in_notebook():
         run_auto_mode(actor, db, logger)
     else:
         run_interactive_mode(actor, db, logger)
```

Comment:
I think we can run `run_happy_path` function every time (in notebook and in console), when we run pytest or doc building we can setup environment var like `DISABLE_INTERACTIVE_MODE` and check it by `is_interactive_mode` function

Reply:
Add comments, that this conditions are used not to run `run_interactive_mode` function in case when we auto-build notebooks.